Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -2557,6 +2560,24 @@ def visit_CClassDefNode(self, node):
         self.in_py_class = old_in_pyclass
         return node
 
+    def _body_without_directives(self, node, directives_to_remove):
+        # For things like cclass, cfunc, etc the directives
+        # should not keep propagating to nested funcs/classes
+        # after they have been applied. This function creates
+        # a CompilerDirectivesNode to ensure that
+        new_directives = Options.copy_inherited_directives(self.directives)
+        for d in directives_to_remove:
+            del new_directives[d]
+        old_body = node.body
+        if not isinstance(node.body, Nodes.StatListNode):
+            old_body = Nodes.StatListNode(node.pos, stats=[old_body])
```

Comment:
Is this needed? You're not using `old_body` below (probably by accident).

Reply:
No - probably not. It looks like body is expected to be a StatListNode anyway so it should be safe. I'll remove it.